src/Features/CSharp/Portable/CodeCleanup/CSharpCodeCleanupService.cs

```diff
@@ -84,7 +84,8 @@ internal class CSharpCodeCleanupService : ICodeCleanupService
                                        IDEDiagnosticIds.UseExpressionBodyForOperatorsDiagnosticId,
                                        IDEDiagnosticIds.UseExpressionBodyForPropertiesDiagnosticId,
                                        IDEDiagnosticIds.UseExpressionBodyForIndexersDiagnosticId,
-                                       IDEDiagnosticIds.UseExpressionBodyForAccessorsDiagnosticId)),
+                                       IDEDiagnosticIds.UseExpressionBodyForAccessorsDiagnosticId,
+                                       IDEDiagnosticIds.UseExpressionBodyForLocalFunctionsDiagnosticId)),
```

Comment:
>UseExpressionBodyForLocalFunctionsDiagnosticId [](start = 56, length = 46)

If you don't mind, could you share a screenshot to show what the options look like?

Reply:
Sure:
![image](https://user-images.githubusercontent.com/30838294/50164920-2353bc00-02e4-11e9-92d0-b462f11ed759.png)